web/handlers/http.go

```diff
@@ -152,11 +154,26 @@ func New(
 		auth.SetStore(store),
 		auth.SetErrorHandler(authErrH),
 		auth.SetNotAuthorizedHandler(notAuthorizedH),
+		auth.SetLifetime(2*time.Hour), // TODO: configure
```

Comment:
@staltz this was also why we were seeing these short session times.

open to better defaults.

Reply:
Makes sense!